### ISSUE TITLE
Update Pop-Up not keyboard Accessible

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#34198]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

When an update is available and the pop-up appears when you open MuseScore it is not possible to get to the Close button in order to begin the update with keyboard navigation.

STEPS:
1.  Open MuseScore 4.7.3 or earlier.  The Pop-Up appears to offer the update to 4.7.4.
2. Press Enter to accept this.  The focus is on the correct button even though this pop-up can not be navigated to read text or activate the links.
3. After a few seconds an information dialog appears which says MuseScore must Close in order to update and announces the Close button.
4. Press Enter to Close and the pop-up disappears.  You are on an empty MuseScore workspace and the update does not progress.

It looks like after running OCR on this screen that there are in fact two buttons, Close and Cancel.  The Cancel button is not announced by the screen reader and nothing on that screen can be navigated.  The focus must be on the Cancel button, so every time when Enter is pressed the update is cancelled and the user cannot make it begin or work out what the issue is.

This was working correctly in earlier versions of MuseScore 4, possibly 4.6.

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.7.4

### Regression

Yes, this used to work in a previous version of MuseScore 4.x

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by jmwmusic** [comment]
I've just struck this same issue trying to update to 4.7.5 and couldn't remember the work around myself.  Most blind users are not working out how to get around this and having to download updates from the website.  Making arrows work to move between these two buttons and the Cancel that is in focus read when the pop-up comes up I would have thought is fairly straight forward.  Could this be given some priority before the next update comes out?

**Comment by jmwmusic** [comment]
Some users are reporting that they can access this by pressing an up arrow followed by enter.  So, it looks like the keyboard navigation does work and like some other dialogs the speech is missing with NVDA.  Note that when NVDA reads out the whole dialog when it opens it says "Close button" at the end, but does not speak the "Cancel button" even though that is where focus is.  When arrows are pressed there is no speech feedback of any sort.
